client/me/purchases/manage-purchase/index.jsx

```diff
@@ -646,10 +647,42 @@ class ManagePurchase extends Component {
 					</>
 				);
 			}
-
 			return description;
 		}
 
+		if ( isDIFMProduct( purchase ) ) {
+			const productDetails = productsList[ WPCOM_DIFM_LITE ];
+			const [ tier0 ] = productDetails.priceTierList;
```

Comment:
This also gives this error since the product details might not have been loaded from the server:
<img width="633" alt="image" src="https://user-images.githubusercontent.com/5436027/187842462-f6fa5695-9f0a-4cd5-b2a6-6b842a94b6bb.png">

Reply:
Sorry I thought I took care of this 😬